vysledek/style.css

```diff
@@ -0,0 +1,327 @@
+*, ::after, ::before { box-sizing: border-box; }
+
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #f5f5f5;
+    font-family: 'Merriweather', serif;
+    font-size: 16px;
+    line-height: 1.6;
+}
+
+.leading {
+    display: flex;
+    flex-basis: 100%;
+    flex-direction: column;
+    font-family: 'Raleway', sans-serif;
+    font-weight: 600;
+    justify-content: space-around;
+    min-width: 440px;
+    background-color: white;
+
+}
+
+/*                  HEADER                              */
+header {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    background-image: url("/images/island-hlavicka.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    color: white;
+    text-align: center;
+    font-weight: 700;
```

Comment:
⚠Pozor na to, že `font-weight` se dědí a proto jsou všechny texty ve hlavičce tučně. To podle zadání úplně u všech nechceme.